app/src/views/About.vue

```diff
@@ -112,67 +112,147 @@
+          </p>
+          
           <p class="mt-6 font-bold">
-            How long does it take for content to be approved and appear on the
+            How long does it take for content to appear on the
             site?
           </p>
           <p class="mt-4">
-            You should be hearing from us within a couple of weeks of having
-            submitted your content. Don’t panic if you don’t hear from us
-            though: the volume of submissions may vary over time, but we’ll make
-            it a point to always get back to you.
+            Once approved, the content gets on the site within a week of approval. 
+          </p>
+
+          <p class="mt-6 font-bold">
+           Can I submit content in different product categories?
           </p>
+          <p class="mt-4">
+            Yes! YOU CAN.
+          </p>
+
           <p class="mt-6 font-bold">
             Is there any reward or recognition for contributors?
           </p>
           <p class="mt-4">
-            If your content is selected, we hope that the increased visibility
-            that your content gets constitutes an extra token of appreciation.
+            What is hard work without any recognition!
+            If you become an author at Dev Library, you receive:
+            <li>
+            <ul> Visibility of your work</ul>
+            <ul> Contributor Badge on Google Developer Profile </ul>
+            <ul> Stamp of Google on your work </ul>
+            <ul> Feature in our monthly newsletter </ul>
+           </li>
+           <p>
+           And many more surprises, on the way.
           </p>
           <p class="mt-6 font-bold">
             How do I request for content to be removed?
           </p>
           <p class="mt-4">
-            Fill out
-            <a href="https://forms.gle/FPQg4s8DUjEdqyQR7" target="blank">
-              this form
+            To remove your content, please raise a Pull Request on the Dev Library’s GitHub repo. 
+            In case you are unable to do this, please fill out the
+            <a href="https://forms.gle/MdMEj7hkdWFLUBrf7" target="blank">
+              Contact Us form
             </a>
             and we will get back to you in 48 hours for next steps.
           </p>
+          
           <p class="mt-6 font-bold">
-            How do I report an error in the technical documentation?
+            How do I raise an issue?
           </p>
           <p class="mt-4">
-            Each piece of content on this site is authored by a different
-            developer and hosted on an external platform (GitHub, Medium, etc).
-            Please use the feedback mechanisms available on those platforms in
-            order to help improve content directly.
+            To raise an issue, please post your issue here on the Dev Library repo.  
+            In case you are unable to do this, please fill out the <a href="https://forms.gle/MdMEj7hkdWFLUBrf7" target="blank"> Contact Us form.
```

Comment:
This `<a>` tag is unclosed. That's the cause of the error below.

Reply:
Thank you Sam!